amr-wind/utilities/sampling/ConeSampler.cpp

```diff
@@ -0,0 +1,189 @@
+#include "amr-wind/utilities/sampling/ConeSampler.H"
+#include "amr-wind/CFDSim.H"
+#include "amr-wind/utilities/tensor_ops.H"
+
+#include "AMReX_ParmParse.H"
+
+#define PI 3.14159265
```

Comment:
shouldn't need to define PI here. pi is available from AMReX by using M_PI, and this is the standard in other portions of the code. you should be able to remove this line and replace PI with M_PI.